building-a-web-application/spec/spec_helper.rb

```diff
@@ -8,7 +8,12 @@
 
 require_relative '../blog'
 
-Sequent::Test::DatabaseHelpers.maintain_test_database_schema(env: 'test')
```

Comment:
Why not keep this actually?

Reply:
It's currently not in the [generated spec_helper.rb](https://github.com/zilverline/sequent/blob/master/lib/sequent/generator/template_project/spec/spec_helper.rb) which is why this was removed/replaced. 

Should I add this change to the generated `spec_helper.rb` instead?

Reply:
Yes, I think we forgot to update that one.

Reply:
[Updated the spec_helper.rb in the Sequent project generator](https://github.com/zilverline/sequent/commit/920bfc42359c96d483b03950250e545151907e8d).